dbt/scripts/utils/export.py

```diff
@@ -389,17 +404,26 @@ def save_model_to_workbook(
                 else sheet[model.start_row : sheet.max_row]
             )
             for row in non_header_rows:
-                for idx, formats in column_format_by_index.items():
-                    for attr, val in formats.items():
-                        if attr == "data_type":
-                            row[idx].value = (
-                                val(row[idx].value)
-                                if row[idx].value != ""
-                                and row[idx].value is not None
-                                else None
-                            )
-                        else:
-                            setattr(row[idx], attr, val)
-
-        print(f"Exported model {model.name} to {output_path}")
-        return output_path
+                for cell in row:
```

Comment:
Iterating every cell in the workbook is very slightly slower than our previous approach, but not enough to make a substantial difference since most of our models are much longer than they are wide.